lab-nassir/lib/add-to-pool.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const pool = require('./pool');
+
+module.exports = function addToPool(client) {
```

Comment:
This file is currently not being used and is causing linter issues. Please remove and resubmit :)